sky/unit/test/widget/stack_test.dart

```diff
@@ -34,10 +35,11 @@ void main() {
       );
 
       Element container = tester.findElementByKey(key);
-      expect(container.renderObject.parentData.top, isNull);
-      expect(container.renderObject.parentData.right, isNull);
-      expect(container.renderObject.parentData.bottom, isNull);
-      expect(container.renderObject.parentData.left, equals(10.0));
+      StackParentData parentData = container.renderObject.parentData;
```

Comment:
declare it earlier so that the code here has the same indentation as the later use